types/audit-details.d.ts

```diff
@@ -167,6 +167,15 @@ declare global {
         nodeLabel?: string;
       }
 
+      export interface SourceLocationValue {
+        type: 'source-location';
+        /** False if url doesn't refer to a real network resource - which is when there's a magic comment sourceURL. */
+        canLink: boolean;
```

Comment:
WDYT about....

- `linksToNetworkResource`
- `urlIsNetworkResource`
- `isSourceUrlHrefLinkable`
- something else?

:)

Reply:
I like `urlIsNetworkResource`. Would remove half the comment here too.